salt/utils/__init__.py

```diff
@@ -2563,7 +2562,11 @@ def is_dictlist(data):
     return False
 
 
-def repack_dictlist(data):
+def repack_dictlist(data,
```

Comment:
@terminalmage, @ticosax, this change broke [`unit.states.dockerng_test.DockerngTestCase.test_running_discard_wrong_environment_values`](https://jenkins.saltstack.com/job/branch_tests/job/develop/job/salt-linode-cent6/1350/testReport/junit/unit.states.dockerng_test/DockerngTestCase/test_running_discard_wrong_environment_values/).  It is not clear to me what the best option is to fix the test, so would either of you mind looking into it?  Thanks.